torchvision/datapoints/_datapoint.py

```diff
@@ -217,51 +215,54 @@ def perspective(
     ) -> Datapoint:
         return self
 
-    def elastic(
+    def _elastic(
         self,
         displacement: torch.Tensor,
         interpolation: Union[InterpolationMode, int] = InterpolationMode.BILINEAR,
         fill: _FillTypeJIT = None,
     ) -> Datapoint:
         return self
 
-    def rgb_to_grayscale(self, num_output_channels: int = 1) -> Datapoint:
+    def _rgb_to_grayscale(self, num_output_channels: int = 1) -> Datapoint:
         return self
 
-    def adjust_brightness(self, brightness_factor: float) -> Datapoint:
+    def _adjust_brightness(self, brightness_factor: float) -> Datapoint:
         return self
 
-    def adjust_saturation(self, saturation_factor: float) -> Datapoint:
+    def _adjust_saturation(self, saturation_factor: float) -> Datapoint:
         return self
 
-    def adjust_contrast(self, contrast_factor: float) -> Datapoint:
+    def _adjust_contrast(self, contrast_factor: float) -> Datapoint:
         return self
 
-    def adjust_sharpness(self, sharpness_factor: float) -> Datapoint:
+    def _adjust_sharpness(self, sharpness_factor: float) -> Datapoint:
         return self
 
-    def adjust_hue(self, hue_factor: float) -> Datapoint:
+    def _adjust_hue(self, hue_factor: float) -> Datapoint:
         return self
 
-    def adjust_gamma(self, gamma: float, gain: float = 1) -> Datapoint:
+    def _adjust_gamma(self, gamma: float, gain: float = 1) -> Datapoint:
         return self
 
-    def posterize(self, bits: int) -> Datapoint:
+    def _posterize(self, bits: int) -> Datapoint:
         return self
 
-    def solarize(self, threshold: float) -> Datapoint:
+    def _solarize(self, threshold: float) -> Datapoint:
         return self
 
-    def autocontrast(self) -> Datapoint:
+    def _autocontrast(self) -> Datapoint:
         return self
 
-    def equalize(self) -> Datapoint:
+    def _equalize(self) -> Datapoint:
         return self
 
-    def invert(self) -> Datapoint:
+    def _invert(self) -> Datapoint:
         return self
 
-    def gaussian_blur(self, kernel_size: List[int], sigma: Optional[List[float]] = None) -> Datapoint:
+    def _gaussian_blur(self, kernel_size: List[int], sigma: Optional[List[float]] = None) -> Datapoint:
+        return self
+
+    def _normalize(self, mean: List[float], std: List[float], inplace: bool = False) -> Datapoint:
```

Comment:
This was missed in #7113. We technically don't need it, since both the dispatcher

https://github.com/pytorch/vision/blob/08c9938f3e69b4fb2d710a038479264f3c23b133/torchvision/transforms/v2/functional/_misc.py#L65-L70

and the transform

https://github.com/pytorch/vision/blob/08c9938f3e69b4fb2d710a038479264f3c23b133/torchvision/transforms/v2/_misc.py#L162

filter on the datapoint type. However, this is the only dispatcher that does not have a method defined on the datapoint base class.